examples/quickstart-chrome/src/test/java/org/fluentlenium/examples/test/chromium/DuckDuckGoChromiumApiTest.java

```diff
@@ -0,0 +1,52 @@
+package org.fluentlenium.examples.test.chromium;
+
+import com.google.common.collect.ImmutableMap;
+import org.fluentlenium.adapter.junit.FluentTest;
+import org.fluentlenium.utils.ChromiumApi;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.Response;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class DuckDuckGoChromiumApiTest extends FluentTest {
+
+    private ChromiumApi chromium;
+    private Response response;
+
+    @Override
+    public WebDriver newWebDriver() {
+        return new ChromeDriver();
+    }
+
+    @Before
+    public void setUp() {
+        ChromeDriver driver = (ChromeDriver) getDriver();
+        chromium = getFluentControl().makeChromiumApi(driver);
+    }
+
+    @Test
+    public void resultPageUrlShouldContainSearchQueryName() {
+        String searchPhrase = "searchPhrase";
+        String duckDuckUrl = "https://duckduckgo.com";
+
+        chromium.sendCommand("Page.navigate", ImmutableMap.of("url", duckDuckUrl));
+        chromium.sendCommand("Input.insertText", ImmutableMap.of("text", searchPhrase));
+        chromium.sendCommand("Input.dispatchKeyEvent", sendEnterKeyEventParams());
```

Comment:
All of those three lines should start with getChromiumApi().sendCommand(

Reply:
Fixed.